main-support.ts

```diff
@@ -590,17 +578,28 @@ function extractMetadataForThisONEFile(
       const origWidth = metadata.streams[0].width;
       const origHeight = metadata.streams[0].height;
       const sizeLabel = labelVideo(origWidth, origHeight);
-      const fileSize = metadata.format.size;
+      const fileSize: string = metadata.format.size; // looks like a number, but actually a string!
```

Comment:
Is this intentional? ie. is there a reason we have it as a string?

Reply:
It turns out that `ffprobe` outputs `format.size` as a string (I checked with `console.log(typeOf(metadata.format.size))` -- so I added the type definition to track what happens to it 👍

Reply:
Ha! - Should we parse the integer here?

Reply:
Makes sense -- will do 👍